db/data/20230306222313_change_f13614c_document_type_on_documents.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class ChangeF13614cDocumentTypeOnDocuments < ActiveRecord::Migration[7.0]
```

Comment:
When I ran `rake data:migrate` locally it failed to migrate "ArchiveOnVisaColumns" because "PG::UndefinedColumn: ERROR:  column "was_on_visa" does not exist." I had to remove both the archived data migrations to test if this migration worked. Hopefully we won't have this issue on prod and demo b/c it was already run on there?